src/global.d.ts

```diff
@@ -1,22 +0,0 @@
-// The ResizeObserver API is available in all major browsers. However,
-// TypeScript has yet to implement any type declarations for it. A basic
-// incomplete version of the API is declared here just to get the compiler
-// not to throw errors.
-//
-// https://github.com/microsoft/TypeScript/issues/37861
-
-interface IResizeObserver {
```

Comment:
Unless this change is visible to consumers in any way, I think this is a `refactor` rather than `fix`.

Unless you specifically want the version bump, then I guess leave it 😄

Reply:
I made it a `fix` because I got build errors I think :slightly_smiling_face: